tool_collections/cufflinks/cuffdiff/cuffdiff_wrapper.xml

```diff
@@ -287,7 +286,7 @@
                 <param name="do_normalization" value="No" />
                 <param name="multiread_correct" value="No"/>
                 <param name="sAdditional" value="No"/>
-                <output name="output_cummerbund" ftype="sqlite" file="cuffdiff_out.sqlite" compare="sim_size" />
+                <output name="output_cummerbund" ftype="sqlite" file="cuffdiff_out.sqlite" compare="sim_size" delta="20000" />
```

Comment:
Maybe you don't need to increase the delta if you update cuffdiff_out.sqlite, which I think it's fair since bioconductor-cummerbund version has been increased.